tools/tools.go

```diff
@@ -0,0 +1,18 @@
+//go:build tools
+// +build tools
+
+package tarod
+
+// The other imports represent our build tools. Instead of defining a commit we
+// want to use for those golang based tools, we use the go mod versioning system
+// to unify the way we manage dependencies. So we define our build tool
+// dependencies here and pin the version in go.mod.
+import (
+	_ "github.com/btcsuite/btcd"
+	_ "github.com/dvyukov/go-fuzz/go-fuzz"
```

Comment:
I think we only need `golangci-lint`, `go-acc` and `gosimports` here, should clean up the go.mod somewhat.